src/components/dashboard/SendLinkSummary.js

```diff
@@ -32,8 +33,34 @@ const SendLinkSummary = (props: AmountProps) => {
   const generateLink = async () => {
     setLoading(true)
     try {
-      const sendLink = await goodWallet.generateLink(amount)
-      await API.sendLinkByEmail(to, sendLink)
+      // Generate link deposit
+      const { sendLink, receipt } = await goodWallet.generateLink(amount)
+
+      // Save transaction
+      const transactionEvent: TransactionEvent = {
+        id: receipt.blockHash,
+        date: new Date().toString(),
+        type: 'send',
+        data: {
+          to,
+          reason,
+          amount,
+          sendLink,
+          receipt
+        }
+      }
+      await UserStorage.updateFeedEvent(transactionEvent)
+
+      // Cheking events are being stored
```

Comment:
We don't need to reread the day events. maybe catch any exceptions from updatefeedevent.
Maybe also as a security step for not losing the deposit code if any exception (in updating feed/sending email/sms) then withdraw the deposit and notify user

Reply:
Abut reading events. I get that we don't need to do this here. Just put that code there to show that this event is being added since we don't have event feed yet. Should I remove those lines?

About catching errors as a security step: will do